tests/assertions.py

```diff
@@ -205,8 +206,9 @@ def assert_minidump(envelope):
     assert minidump.payload.bytes.startswith(b"MDMP")
 
 
-def assert_timestamp(ts, now=datetime.now(UTC)):
-    assert ts[:11] == now.isoformat()[:11]
+def assert_timestamp(ts):
+    elapsed_time = datetime.now(timezone.utc) - datetime.fromisoformat(ts)
```

Comment:
Had to add the timezone to avoid `TypeError: can't subtract offset-naive and offset-aware datetimes`. I think the subtraction still works since it [converts both to utc](https://docs.python.org/3/library/datetime.html#:~:text=input%20is%20aware.-,Subtraction,-of%20a%20datetime)

Reply:
You could have just used the `now` parameter, right?

Reply:
right, removed the unnecessary imports & moved the `now` into the function body to evaluate it for every call